deploy/gcp/manifests/local-volume-provisioner.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
Is this file the same as `../../../manifests/local-dind/local-volume-provisioner.yaml`? If so, I think it's better to symlink there.

Reply:
The local-dind one doesn't have the tolerations. Having said that, I did copy the one from the AWS deploy, maybe we should move it to a common directory and symlink from both?